### PR TITLE
Playlist search

### PR DESCRIPTION
**Pull request by leojcollard**
This implements the Spotify Playlist Search API functionality. It was requested in issue #30 and is relevant to a personal project of mine. I followed your code structure to implement this functionality and have included tests.

### CODE REVIEW

**Comment by ChrisCooney** [comment]
Is it possible that this could be merged into the master branch and the maven resource be updated? This feature would be awesome.

File: src/test/java/com/wrapper/spotify/methods/PlaylistSearchRequestTest.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+		List<SimplePlaylist> playlists = playlistSearchResult.getItems();
+		assertEquals(7, playlists.size());
+
+		SimplePlaylist firstPlaylist = playlists.get(0);
+		assertEquals("http://open.spotify.com/user/theodorduf/playlist/5W11lXKiV9B5RZluPrxmCS",
+				firstPlaylist.getExternalUrls().get("spotify"));
+		assertEquals("https://api.spotify.com/v1/users/theodorduf/playlists/5W11lXKiV9B5RZluPrxmCS",
+				firstPlaylist.getHref());
+		assertEquals("5W11lXKiV9B5RZluPrxmCS", firstPlaylist.getId());
+		assertEquals("Tdog✌🏿️", firstPlaylist.getName());
```
Comment (Dargmuesli) [inline src/test/java/com/wrapper/spotify/methods/PlaylistSearchRequestTest.java]: This line throws an `error: unmappable character for encoding Cp1252`

Comment (leojcollard) [inline src/test/java/com/wrapper/spotify/methods/PlaylistSearchRequestTest.java]: Good catch! I have removed that character. Let me know if you find anything else!

**Comment by dr-carrot** [comment]
Is this going to be a thing? It would be really useful to me (especially since I had expected this to merged.)

**Comment by Dargmuesli** [comment]
I've added the `invalid` tag because this feature was merged into branch `v2_requests`, but not into the `master` branch yet. This way I have a better overview over all PRs.

**Pull request merged**